docs/developer/index.rst

```diff
@@ -12,16 +12,16 @@ Before you submit a pull request to Bodhi, please ensure that it meets these cri
 
 * All tests must pass.
 * New code must have 100% test coverage. This one is particularly important, as we don't want to
-  deploy any broken code into production. After you've run ``btest``, you can verify your new code's
-  test coverage with ``bdiff-cover`` in the Vagrant environment, or
-  ``diff-cover coverage.xml --compare-branch=origin/develop --fail-under=100`` if you are not using
-  Vagrant.
```

Comment:
Why drop the documentation on how to run diff-cover for non-Vagrant users?

Reply:
I thougt the btest is also working outside Vagrant.

Reply:
Nah it's a bash alias setup in vagrant